maps/src/main/java/org/odk/collect/maps/layers/OfflineMapLayersViewModel.kt

```diff
@@ -38,7 +38,7 @@ class OfflineMapLayersViewModel(
         _isLoading.value = true
         scheduler.immediate(
             background = {
-                val layers = referenceLayerRepository.getAll()
+                val layers = referenceLayerRepository.getAll().sortedBy { it.name }
```

Comment:
We need to call `loadExistingLayers` in `deleteLayer` to sort there as well right? I guess maybe deletion would never cause a problem because the list is already sorted?

Reply:
It should keep the order I've added a test https://github.com/getodk/collect/pull/6188/commits/f7a5db5c4fcf2387e21ae8468f1795586b09793a